content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -104,46 +102,70 @@ generator client {
 
 Then you can pass an async function into [`$transaction`](../../../guides/performance-and-optimization/prisma-client-transactions-guide#transaction-api).
 
-The following is an example of purchasing a movie ticket online. In this example, we really don't want to sell the same ticket twice:
+Let's say we're building an online banking system. One of the actions we want to perform is to send money from one person to another person.
+
+As experienced developers, we want to make sure that during the transfer,
```

Comment:
Not sure about the "As experienced developers". I'd keep it dry with "You want to ensure the following guarantees:" or something like that